src/backend/app.js

```diff
@@ -21,10 +22,86 @@ app.use(cors());
 
 router.use("/meals", mealsRouter);
 
+//Respond with all meals in the future (relative to the when datetime) - /future-meals
+
+app.get("/future-meals", async (req, res) => {
+  try {
+    const today = new Date();
```

Comment:
since you have used this constant 2 times in your code it's better to put it out of the block and reuse it